src/admin/dumb_components/letter_body.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+export default ({ letter }) => {
+
+  const formatContent = (content) => (
+    <div>
+      {content.split('\n').map((paragraph, i) =>
```

Comment:
I think leave it as at is for now.

The problem with setting the key to the index in some cases is that the whole point of the key is for React to know when 2 elements being iterated over are the same. If, for example, you're getting data from AJAX requests or the data are otherwise dynamic, using the key as the index will actually cause its state to be lost -- including any state kept outside of React.

It's not an issue in this case